src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs

```diff
@@ -0,0 +1,443 @@
+                ht.Add("System.Security.Cryptography.SHA256", SHA256DefaultType);
+
+                ht.Add("SHA384", SHA384DefaultType);
+                ht.Add("SHA-384", SHA384DefaultType);
+                ht.Add("System.Security.Cryptography.SHA384", SHA384DefaultType);
+
+                ht.Add("SHA512", SHA512DefaultType);
+                ht.Add("SHA-512", SHA512DefaultType);
+                ht.Add("System.Security.Cryptography.SHA512", SHA512DefaultType);
+
+                // Keyed Hash Algorithms
+                ht.Add("System.Security.Cryptography.HMAC", HMACSHA1Type);
+                ht.Add("System.Security.Cryptography.KeyedHashAlgorithm", HMACSHA1Type);
+                ht.Add("HMACMD5", HMACMD5Type);
+                ht.Add("System.Security.Cryptography.HMACMD5", HMACMD5Type);
+                ht.Add("HMACSHA1", HMACSHA1Type);
+                ht.Add("System.Security.Cryptography.HMACSHA1", HMACSHA1Type);
+                ht.Add("HMACSHA256", HMACSHA256Type);
+                ht.Add("System.Security.Cryptography.HMACSHA256", HMACSHA256Type);
+                ht.Add("HMACSHA384", HMACSHA384Type);
+                ht.Add("System.Security.Cryptography.HMACSHA384", HMACSHA384Type);
+                ht.Add("HMACSHA512", HMACSHA512Type);
+                ht.Add("System.Security.Cryptography.HMACSHA512", HMACSHA512Type);
+
+                // Asymmetric algorithms
+                ht.Add("RSA", RSACryptoServiceProviderType);
+                ht.Add("System.Security.Cryptography.RSA", RSACryptoServiceProviderType);
+                ht.Add("System.Security.Cryptography.AsymmetricAlgorithm", RSACryptoServiceProviderType);
+
+                ht.Add("DSA", DSACryptoServiceProviderType);
+                ht.Add("System.Security.Cryptography.DSA", DSACryptoServiceProviderType);
+
+                ht.Add("ECDsa", ECDsaCngType);
+                ht.Add("ECDsaCng", ECDsaCngType);
+                ht.Add("System.Security.Cryptography.ECDsaCng", ECDsaCngType);
+
+                // Symmetric algorithms
+                ht.Add("DES", DESCryptoServiceProviderType);
+                ht.Add("System.Security.Cryptography.DES", DESCryptoServiceProviderType);
+
+                ht.Add("3DES", TripleDESCryptoServiceProviderType);
+                ht.Add("TripleDES", TripleDESCryptoServiceProviderType);
+                ht.Add("Triple DES", TripleDESCryptoServiceProviderType);
+                ht.Add("System.Security.Cryptography.TripleDES", TripleDESCryptoServiceProviderType);
+
+                ht.Add("RC2", RC2CryptoServiceProviderType);
+                ht.Add("System.Security.Cryptography.RC2", RC2CryptoServiceProviderType);
+
+                ht.Add("Rijndael", RijndaelManagedType);
+                ht.Add("System.Security.Cryptography.Rijndael", RijndaelManagedType);
+                // Rijndael is the default symmetric cipher because (a) it's the strongest and (b) we know we have an implementation everywhere
```

Comment:
I imagine you copied this from full framework. We might want to drop this comment since it's decidedly outdated.